cmd/disk-cache.go

```diff
@@ -806,7 +793,7 @@ func newCache(c CacheConfig) (*diskCache, error) {
 	var cfsObjects []*cacheFSObjects
 	formats, err := loadAndValidateCacheFormat(c.Drives)
 	if err != nil {
-		errorIf(err, "Cache drives validation error")
+		return nil, err
```

Comment:
returning err here instead of logging and continuing will prevent using disk cache if just one drive is down/added to configuration. I thought we had agreed that we would allow degraded behavior by using cache with remaining drives